.circleci/config.yml

```diff
@@ -143,7 +143,7 @@ jobs:
           working_directory: packages/contracts-bedrock
       - run:
           name: check go bindings
-          command: make && git diff --exit-code
+          command: make -j 8 && git diff --exit-code
```

Comment:
8x parallel, speeds up bindings by ~4x on my machine.

Reply:
This check is pretty slow, recent foundry commit https://github.com/foundry-rs/foundry/commit/cca65b1fc6c7ab50b5c84c99c9d3537cf00dbbbb should speed it up.

Open to having this as a separate PR, altho we should make `8` be dynamic based on the # of cores

Reply:
Yeah I had to revert this because -j 8 didn't work on CI